tests/tests-core/src/main/java/com/gentics/mesh/core/node/NodeEndpointTest.java

```diff
@@ -370,7 +369,7 @@ public void testCreateDeleteCreateWithUuid() throws Exception {
 	@Test
 	@Override
 	public void testCreateWithDuplicateUuid() throws Exception {
-		String nodeUuid = tx(() -> project().getUuid());
+		String nodeUuid = tx(() -> project().getBaseNode().getUuid());
```

Comment:
Since we cannot prove that the UUID is globally unique, without enumerating all the existing data, we have to change the test to check the current entity only.